client.go

```diff
@@ -782,3 +784,35 @@ func CheckRuntime(current, expected string) bool {
 	}
 	return true
 }
+
+func (c *Client) GetSnapshotterSupportedPlatforms(ctx context.Context, snapshotterName string) (platforms.MatchComparer, error) {
+	filters := []string{fmt.Sprintf("type==%s, id==%s", plugin.SnapshotPlugin, snapshotterName)}
+	in := c.IntrospectionService()
+	log.G(ctx).WithField("in", in).Info("client introspection service")
```

Comment:
Should we log snapshotter supported platforms at start and not log it here? I'm concerned if this is called a lot its extra traces that dont change in the lifetime of the daemon

Reply:
Good point. I removed the log statements here.